test/nl/surf/eduhub_rio_mapper/smoke_test.clj

```diff
@@ -129,10 +134,27 @@
                            [8 "delete" :eduspec  eduspec-parent-id goedgekeurd?]
                            [9 "upsert" :course   course-id         #(= (-> % :errors :message)
                                                                        "No education specification found with id: fddec347-8ca1-c991-8d39-9a85d09cbcf5")]]]
+    ;; Test with http message logging enabled
+    (let [[idx action ootype id pred?] [1 "upsert" :eduspec  eduspec-parent-id goedgekeurd?]]
+      (testing (str "Command " idx " " action " " id)
+        (binding [http-utils/*vcr* (vcr idx (str action "-" (name ootype)))]
+          (let [result        (logging-runner ootype id action)
+                http-messages (:http-messages result)
+                oplcode       (-> result :aanleveren_opleidingseenheid_response :opleidingseenheidcode)]
+            (when oplcode (swap! code #(if (nil? %) oplcode %)))
+            (when (= 1 idx)
+              (is (= "https://vt-webservice.duo.nl:6977/RIO/services/raadplegen4.0" (some-> http-messages (nth 1 nil) :req :url)))
+              (is (= 200 (some-> http-messages (nth 1 nil) :res :status))))
+            (is (pred? result) (str action "-" (name ootype) idx))))))
+
     (doseq [[idx action ootype id pred?] commands]
       (testing (str "Command " idx " " action " " id)
         (binding [http-utils/*vcr* (vcr idx (str action "-" (name ootype)))]
          (let [result  (runner ootype id action)
+               http-messages (:http-messages result)
                oplcode (-> result :aanleveren_opleidingseenheid_response :opleidingseenheidcode)]
            (when oplcode (swap! code #(if (nil? %) oplcode %)))
+           (when (= 1 idx)
+             (is (nil? (some-> http-messages (nth 1 nil) :req :url)))
+             (is (nil? (some-> http-messages (nth 1 nil) :res :status))))
```

Comment:
`http-messages` is hier altijd nil. dus `(is (nil? http-messages))` is een betere test